### ISSUE TITLE
Impossible Dice as Icon

### ISSUE THREAD

**Issue by RandalR** [BdR76/RandomValuesNPP#18]
I was looking at my Notepad++ toolbar this eve, and noticed that the icon used for the Random Values plug-in showed a die that doesn't (usually) exist.  It shows a die with a ⚀ (one) on top, a ⚃ (four) below and to the left, and a ⚂ (three) next to it.

The standard die is going to have every two opposing sides add up to **7**.  So the ⚀ (one) side will have the ⚅ (six) opposite it, the ⚁ (two) will have the ⚄ (five) opposite it, and the ⚂ (three) will have the ⚃ (four) opposite it.

There is another configuration to keep in mind:  there are two ways of accomplishing the above layout:
-  it is called a "Right-handed" die if starting at the  ⚀ (one) side and moving counter-clockwise, the ⚁ (two) side and then the ⚂ (three) are in order.  This looks like this: 🎲
- it is called a "Left-handed" die if starting at the ⚀ (one) side and moving clockwise, the ⚁ (two) side and then the ⚂ (three) are in that order.

Western dice are right-handed, Chinese dice are left-handed.  Here is a graphic about handedness: 
![diehanded_sm](https://github.com/BdR76/RandomValuesNPP/assets/34193947/ea06b17e-ed3b-4434-adaa-12f15aa92d4f)

I've fixed the icons and they are in here: 
[dice_black_32px.zip](https://github.com/BdR76/RandomValuesNPP/files/13403175/dice_black_32px.zip)
it replaces this file in the project:  RandomValuesNppPlugin/Resources/dice_black_32px.ico

and in here:
[dice_white_32px.zip](https://github.com/BdR76/RandomValuesNPP/files/13403176/dice_white_32px.zip)
it replaces this file in the project:  RandomValuesNppPlugin/Resources/dice_white_32px.ico

I'm sorry to attach the files that way, but I'm a rank amateur when it comes to GitHub and how to do things properly, and I might have known at one time, but have forgotten how to do everything except log in.  I do hope this is helpful.

**Comment by BdR76** [comment]
Ha that's funny, you're absolutely right. 😀 Back when I added it, I just searched for an existing die icon (I think [from here](https://www.shutterstock.com/search/dice-icon?image_type=vector) maybe) and slightly changed the line thickness and color and converted it to .ico size. I never noticed it was an incorrect die, because of the 4 and 3 next to each other.

Btw if you do a [google image search](https://www.google.com/search?q=dice+icon&tbm=isch) there are lots of "faulty" dice icons, 1 and 6 next to each other, the same number twice etc.

Anyway I've fixed it and I'll create a new release, along with some other minor bugs fixes

**Comment by RandalR** [comment]
Hi there, thanks so much for getting back to me!

 

You’re right, there are a lot of incorrect dice icons out there, in fact I didn’t know there was such a thing as right-handed vs. left-handed dice a few years ago despite being an aficionado of dice and of TTRPGs in general.  It wasn’t really until I had to make some dice icons for an app I was working on and fell down a bit of a rabbit hole.

 

I also recently learned that there is yet another way of putting numbers on dice called a “spindown” configuration.  From Wikipedia <https://en.wikipedia.org/wiki/Dice#Common_variations> :

 

Some twenty-sided dice have a different arrangement used for the purpose of keeping track of an integer that counts down, such as health points. These spindown dice are arranged such that adjacent integers appear on adjacent faces, allowing the user to easily find the next lower number. They are commonly used with collectible card games <https://en.wikipedia.org/wiki/Collectible_card_game> .

 

Though those I think are used with d20 and higher.

 

If more folks new that the opposing sides of 6-sided dice must add up to 7, there would be a lot fewer bad images out there lol.  I think that rule is also kept for other n-hedrons that have opposing sides… that Wikipedia article as a neat list.  However, The Dice Lab <http://thedicelab.com/>  and The Dice Shop <https://www.mathartfun.com/DiceShop.html>  have what I consider to be the definitive list and best place to purchase dice from!!

 

I’m happy that I was able to contribute, despite my lack of GitHub savvy, and will smile every time I go to click on the new icon!  Cheers!

 

               -Randal R.
